docs/eip/5559.md

```diff
@@ -0,0 +1,447 @@
+---
+ERC: "5559"
+TITLE: "Cross-Chain Write Deferral Protocol"
+DESCRIPTION: Cross-chain write deferral protocol incorporating secure write deferrals to L2s and centralised databases
```

Comment:
Please retain original fields like the description unless there's a compelling need to change them.

Reply:
Minor change in description is required to remove the word 'mutation', since it is not used in the new version